src/Components/Nav/NavHeadLogos.js

```diff
@@ -1,13 +1,16 @@
 import React, { Component } from 'react';
 import { BsBell, BsList } from 'react-icons/bs';
 import { CgShoppingCart } from 'react-icons/cg';
+import { Link } from 'react-router-dom';
 import './Nav.scss';
 
 class Headlogos extends Component {
   render() {
     return (
       <div className="headLogos">
-        <div className="topLogo">DIENSTAG</div>
+        <Link to={'/'}>
```

Comment:
```suggestion
        <Link to='/'>
```
그냥 스트링 넣어주실 때는 중괄호 불필요합니다!